src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,8 +2,16 @@
 
 ## Unreleased
 
-* Introduced `RuntimeContext` API
-  ([#948](https://github.com/open-telemetry/opentelemetry-dotnet/pull/948))
+* Introduced `RuntimeContext` API.
+  ([#948](https://github.com/open-telemetry/opentelemetry-dotnet/pull/948)).
+  Link constructor changed to accept ActivityTagsCollection instead of
+  IDictionary<string, object> attributes. TelemetrySpan adds more overloads for
+  SetAttribute with value of type bool, int, double. (string already existed).
+  TelemetrySpan's SetAttribute behavior changed to match the
+  [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-attributes).
+    Setting an attribute with an existing key now results in overwriting it.
+    Setting null value has no impact except if null is set to an existing key,
+    it gets removed.
```

Comment:
```suggestion
* Link constructor changed to accept ActivityTagsCollection instead of
  IDictionary<string, object> attributes. TelemetrySpan adds more overloads for
  SetAttribute with value of type bool, int, double (string already existed).
  TelemetrySpan's SetAttribute behavior changed to match the
  [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-attributes).
  Setting an attribute with an existing key now results in overwriting it.
  Setting null value has no impact except if null is set to an existing key,
  it gets removed.
  ([#954](https://github.com/open-telemetry/opentelemetry-dotnet/pull/954)).
```